--- /var/folders/v2/db7y7prn7rz299rl3188lh6c0000gn/T/tmp.oLN0IlCiZS	2026-05-07 14:16:16
+++ ./.config/starship.toml	2026-05-07 14:12:49
@@ -22,15 +22,39 @@
 $python\
 [](fg:green bg:sapphire)\
 $conda\
-[](fg:sapphire bg:lavender)\
+[](fg:sapphire bg:yellow)\
+$aws\
+[](fg:yellow bg:lavender)\
 $time\
 [ ](fg:lavender)\
+$jobs\
+$battery\
 $cmd_duration\
 $line_break\
 $character"""
 
 palette = 'catppuccin_mocha'
 
+command_timeout = 900
+
+[battery]
+disabled = false
+unknown_symbol = '󰂎 '
+
+[[battery.display]]
+threshold = 20
+style = 'bold red'
+
+[aws]
+disabled = false
+format = '[ $symbol($profile) (\($region\) )]($style)'
+symbol = '☁️ '
+force_display = false
+style = "bg:yellow fg:crust"
+
+[aws.region_aliases]
+ap-southeast-2 = ''
+
 [os]
 disabled = false
 style = "bg:red fg:crust"
@@ -66,8 +90,9 @@
 [directory]
 style = "bg:peach fg:crust"
 format = "[ $path ]($style)"
-truncation_length = 3
+truncation_length = 16
 truncation_symbol = "…/"
+truncate_to_repo = false
 
 [directory.substitutions]
 "Documents" = "󰈙 "
@@ -89,6 +114,7 @@
 symbol = ""
 style = "bg:green"
 format = '[[ $symbol( $version) ](fg:crust bg:green)]($style)'
+version_format = 'node${major}'
 
 [bun]
 symbol = ""
@@ -114,6 +140,7 @@
 symbol = ""
 style = "bg:green"
 format = '[[ $symbol( $version) ](fg:crust bg:green)]($style)'
+version_format = 'php${major}.${minor}'
 
 [java]
 symbol = " "
@@ -153,7 +180,7 @@
 format = '[[  $time ](fg:crust bg:lavender)]($style)'
 
 [line_break]
-disabled = true
+disabled = false
 
 [character]
 disabled = false
